lib/core/kotekanMode.cpp

```diff
@@ -53,6 +54,9 @@ kotekanMode::~kotekanMode() {
     restServer::instance().remove_get_callback("/pipeline_dot");
     restServer::instance().remove_all_aliases();
 
+    // Dump all trackers before cleanup.
+    KotekanTrackers::instance().dump_trackers();
```

Comment:
I would move this to `kotekan.cpp` and only dump in the case that we exited with an error: see line 495.   I don't think this dump makes sense outside of an error triggered shutdown.